evap/contributor/forms.py

```diff
@@ -45,6 +52,8 @@ def __init__(self, *args, **kwargs):
         self.fields["vote_start_datetime"].localize = True
         self.fields["vote_end_date"].localize = True
 
+        self.fields["participants"].queryset = UserProfile.objects.exclude(is_active=False)
```

Comment:
@janno42 This will give the usual issues with inactive participants not being listed and then removed on save. I guess that is not expected?

Reply:
right, the queryset should also include all already selected users